packages/integrations/src/extraerrordata.ts

```diff
@@ -18,10 +18,18 @@ export class ExtraErrorData implements Integration {
    */
   public name: string = ExtraErrorData.id;
 
+  /** JSDoc */
```

Comment:
not sure what purpose does this have but I saw it the `debug.ts`'s code

Reply:
It’s to get around writing doc strings. We can leave it like this.